Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = [
   "smoke-test",
   "test_utils",
   "xcresult",
+  "test_report",
```

Comment:
This is a separate library (outside of context) to prevent circular dependencies. `cli` depends on `context` and this depends on `cli`.

Reply:
Roger Roger. Wanted to make sure that the upload reuse was fine before committing to splitting out.

Reply:
Going to split this out in a separate PR.